lib/helpers/contacts/Recipients.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * This helper builds a html form and provides a submission endpoint for the form that makes a /contactdb/recipients API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Kraig Hufstedler <kraigory@gmail.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+namespace SendGrid;
+
+class RecipientForm
+{
+    public $html;
+
+    public function __construct($url)
+    {
+        $html = '<form action="' . $url . '" method="post">
+        First Name: <input type="text" name="first-name"><br>
+        Last Name: <input type="text" name="last-name"><br>
+        E-mail: <input type="text" name="email"><br>
+        <input type="submit">
+        </form>';
+        $this->setHtml($html);
+    }
+
+    public function setHtml($html)
```

Comment:
What's the point of having a setter if we're using the constructor already?.

Reply:
Great point- I was trying to follow the pattern in lib/helpers/mail/Mail.php but you're right, it's not necessary. Will remove.